sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.projectKey=fiware-perseo-core-dev-jafernandez
```

Comment:
I understand this connect with some sonar server out there. Could you elaborate on this? Where is that server located? Who is managing it?

If this is a local configuration (i.e. for a local sonar server) it shouldn't be included in perseo master branch.

Reply:
Fixed in 4d08affe85545b26afed5461585bc8bd710d5a85